### PR TITLE
Renamed `layer` types, classes, values and keys from nso to `admin-level-data`

### LINKED ISSUE #222: Rename NSO data -> admin_level_data
This was meant to be done a while ago (as noted [here](https://github.com/WFP-VAM/prism-frontend/issues/29) ) but we didn't get around to it yet. Simply speaking, NSO stands for national statistical office. The type of data we are referring to could come from multiple sources. The definition of this data is that it describes a measurement at the administrative level.

### PR DESCRIPTION
**Pull request by erwagasore**
- [x] Rename all layer types with `nso` to `admin-level-data`
- [x] Rename `NSOLayer` and `NSOLayerProps` to `AdminLevelDataLayer` and `AdminLevelDataLayerProps` respectively
- [x] Test is changes didn't affect the application

### CODE REVIEW

**Comment by wadhwamatic** [comment]
Looks good from my side @erwagasore. You'll need to get the latest from master again. Let's also wait for technical review from @JorgeMartinezG . Let's please aim to get this merged onto master early next week

**Review by JorgeMartinezG — APPROVED** [review]
Good from my side. We would need to merge master before merging, just in case some conflict appears

**Pull request merged**
